### PR TITLE
Supporting multiple htIds in search

### PR DESCRIPTION
**Pull request by mingazhev**
Travelgate channel must support multiple destinations search, added this option in connection with mapper

### CODE REVIEW

File: Api/Models/Availabilities/AvailabilityRequest.cs [hunk 1]
```diff
@@ -14,8 +14,8 @@ namespace HappyTravel.Edo.Api.Models.Availabilities
         [JsonConstructor]
         public AvailabilityRequest(string nationality, string residency, DateTime checkInDate, DateTime checkOutDate,
```
Comment (hasmikharutyunyan) [inline Api/Models/Availabilities/AvailabilityRequest.cs:15]: is this model for our frontend request?

File: Api/Services/Accommodations/Availability/Mapping/AvailabilitySearchAreaService.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CSharpFunctionalExtensions;
+using HappyTravel.Edo.Api.Models.Availabilities.Mapping;
+using HappyTravel.Edo.Api.Models.Locations;
+using HappyTravel.Edo.Common.Enums;
+
+namespace HappyTravel.Edo.Api.Services.Accommodations.Availability.Mapping
+{
+    public class AvailabilitySearchAreaService : IAccommodationMappingService
```
Comment (hasmikharutyunyan) [inline Api/Services/Accommodations/Availability/Mapping/AvailabilitySearchAreaService.cs:10]: Why changed name ?

File: Api/Models/Availabilities/AvailabilityRequest.cs (hunk 1 above)
Comment (mingazhev) [reply Api/Models/Availabilities/AvailabilityRequest.cs:15]: Yes

File: Api/Services/Accommodations/Availability/Mapping/AvailabilitySearchAreaService.cs (hunk 2 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Availability/Mapping/AvailabilitySearchAreaService.cs:10]: In previous version it returned a single LocationDescriptor, info about the location.
In this version I think the data is semantically different, it contains multiple locations.
I've tried to return multiple LocationDescriptors but it was hard to work with them

Comment (hasmikharutyunyan) [reply Api/Services/Accommodations/Availability/Mapping/AvailabilitySearchAreaService.cs:10]: so, maybe needed to change IAccommodationMappingService name also?

Comment (mingazhev) [reply Api/Services/Accommodations/Availability/Mapping/AvailabilitySearchAreaService.cs]: Sure, missed that, now fixed

**Review by hasmikharutyunyan — APPROVED** [review]

**Review by oleg1std — APPROVED** [review]

**Review by oleg1std — APPROVED** [review]

**Pull request merged**